README.md

```diff
@@ -14,3 +14,7 @@ For help and discussion regarding Protobuf plugins, join us on [Slack](https://b
 
 For feature requests, bugs, or technical questions, email us at [dev@buf.build](dev@buf.build).
 
+## Manual action caveats
+
+When triggering a manual execution of the `fetch versions` workflow, you may want to disable the scheduled execution 
```

Comment:
The top-level README is more meant for consumers of the plugins from this repo. We should add this sort of info to CONTRIBUTING.md instead.